bin/analysis.sh

```diff
@@ -17,8 +17,20 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-base_dir=$(dirname "$0")
+# resolve links - $0 may be a softlink (code as used by Tomcat)
+# N.B. readlink would be a lot simpler but is not supported on Solaris
+PRG="$0"
+
+while [ -h "$PRG" ]; do
+	ls=`ls -ld "$PRG"`
+	link=`expr "$ls" : '.*-> \(.*\)$'`
+	if expr "$link" : '/.*' > /dev/null; then
+		PRG="$link"
+	else
+		PRG=`dirname "$PRG"`/"$link"
+	fi
+done
+base_dir=$(dirname "$PRG")
```

Comment:
How about https://github.com/gradle/gradle-project-templates/blob/main/gradlew#L68 ?

Reply:
hi, @D-D-H  Thanks for your comment. I have to say that this approach is better than JMeter's version. I will update the pr later